src/utils/clmfates_interfaceMod.F90

```diff
@@ -930,7 +930,7 @@ subroutine init(this, bounds_proc, flandusepftdat)
             c = this%f2hmap(nc)%fcolumn(s)
             pi = col%patchi(c)+1
             pf = col%patchf(c)
-!            patch%itype(pi:pf) = ispval
+            patch%itype(pi:pf) = ispval
```

Comment:
Add a note about why this is being done, and what it will do.

Reply:
actually, the comments on 925-927 explain what is going on already.

Reply:
let me update the text a little though, its not as clear as it could be

Reply:
updated text, clearer now